src/util/http/UrlParser.h

```diff
@@ -19,21 +19,30 @@ namespace ad_utility {
  */
 class UrlParser {
  public:
+  // A parsed URL.
```

Comment:
Maybe, as now there is not a "class" anymore, 
consider making the `class UrlParser` a `namespace urlParser`.
Then you can add a local `using namespace urlParser` in the tests that otherwise have a lot of `UrlParser::` in them.